qa/suites/rados/singleton-nomsgr/all/ceph-snapmapper.yaml

```diff
@@ -0,0 +1,22 @@
+openstack:
```

Comment:
nit: the commit title should have a space after the component part, so it should look like:

```
osd/SnapMapper: maintain the prefix_itr between calls to SnapMapper::get_next_objects_to_trim()
```

Reply:
Also, there is no need for doubling the `Signed-off-by` tag:

```

Signed-off-by: Gabriel BenHanokh <gbenhano@redhat.com>

typo

Signed-off-by: Gabriel BenHanokh <gbenhano@redhat.com>
```